package/MDAnalysis/coordinates/pdb/__init__.py

```diff
@@ -1,20 +0,0 @@
-# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
Everything under coordinates/pdb/ is gone now? Right?

Add to CHANGELOG a comment that we removed `MDAnalysis.coordinates.pdb` (extensions to the BioPython PDB reader).